src/main/python/hydra/rmqtest/runtest.py

```diff
@@ -63,7 +63,9 @@ def rerun_test(self, options):
                                   msg_requested_rate=self.options.msg_rate)
         l.info("PUB server updated")
 
-        self.reset_all_app_stats(self.rmqsub)
+        # Pass signals in groups of apps
```

Comment:
We need to get the testing to move out of rmq dependencies, 
The base packages requirement for hydra is only on zmq. So the unit testing should exercise zmq tests. 
Additional test's can be run with flags and should run on travis on commits but does not make sense for folks to run rmq when they are testing cassandra.